open-ce/validate_config.py

```diff
@@ -87,4 +87,5 @@ def validate_config(arg_strings=None):
     return 0
 
 if __name__ == '__main__':
+    utils.check_if_conda_build_exists() 
```

Comment:
I think these need to happen before importing build_tree. One other option is to put these checks back in the files that use conda_build so that it is easier to keep track of (you can still use the function call at the top of the file).